include/ros1_bridge/builtin_interfaces_factories.hpp

```diff
@@ -25,11 +23,18 @@
 #include <builtin_interfaces/msg/duration.hpp>
 #include <builtin_interfaces/msg/time.hpp>
 
+#include <memory>
+#include <string>
```

Comment:
Can these two be at the top of the file?

Reply:
they cannot (see description)

Reply:
Sorry, didn't read to closely. :+1: